src/static/css/module/process.less

```diff
@@ -67,58 +67,304 @@
 }
 
 /* temp. workaround for .block-link border-color issue */
- a.jump-link,
- a.list_link {
-   border-color: @link-underline;
-   
-   &:visited {
-     border-color: @link-underline-visited;
-   }
-   
-   &:hover {
-     border-color: @link-underline-hover;
-     z-index: 10;
-   }
+a.jump-link,
+a.list_link {
+ border-color: @link-underline;
+ 
+ &:visited {
+   border-color: @link-underline-visited;
+ }
+ 
+ &:hover {
+   border-color: @link-underline-hover;
+   z-index: 10;
+ }
 
-   &:active{
-     border-color: @link-underline-active;
-   }
+ &:active{
+   border-color: @link-underline-active;
  }
+}
 
 
-.expandable__padded.expandable__flush-sides {
-  margin-right: 0;
-  .expandable_header {
-    padding: unit(10px / @base-font-size-px, em) 0;
-  }
-  .expandable_content {
-    margin: 0;    
-  }
+.icon-round {
+  width: 1.5em;
+  height: 1.5em;
+  border-radius: 1.5em;
+  line-height: 1.5;
+  text-align: center;
+  background-color: @green-tint;
 }
 
-.block__bg.block__small-pad {
-  padding: unit(@grid_gutter-width  / @base-font-size-px, em)
-           unit((@grid_gutter-width / 2) / @base-font-size-px, em);
+.char-icon {
+  position: relative;
+  padding-left: 1.5em;
+ 
+ &:before {
+   position: absolute;
+   left: 0;
+   top: 1px;
+   font-size: .7em;
+   .icon-round;
+   .webfont-medium();
+ }
 }
 
-.list__flush-sides {
-  padding-left: 20px;
+.char-icon__checked {
+ &:before {
+   .cf-icon();
+   content: "\e100";
+   line-height: 1.6;
+ }
+}
+
+.phase_ordinal {
+  .icon-round;
+  color: @black;
+  .webfont-demi();
+  float: left;
+  margin-right: 5px;
 }
 
 /* Process-specific styles */
 
 .know-the-process {
   
+  .process-nav {
```

Comment:
does this need to be nested in `.know-the-process` since you already have a class specific to this nav?